lola-tech/test/features/homepage.feature

```diff
@@ -5,7 +5,77 @@ Feature: Homepage
 
    @test
    Scenario: Access the Homepage
-      Given I access the URL "https://www.lola.tech"
+      Given I access the URL "https://lola-tech.webflow.io/"
       Then The "Hi message" from the "Home" page is displayed
-      And The "Cookie Popup" from the "Home" page is displayed
-      When I click on "Latest Jobs Button" from "Home" page
\ No newline at end of file
+      # And The "Cookie Popup" from the "Home" page is displayed (does not work on stage site)
+      When I click on the "Latest Jobs button" from the "Home" page
+
+   Scenario: Verify Covid banner redirects to correct page
+      Given I access the URL "https://lola-tech.webflow.io/"
+      When I click on the "Covid Banner" from the "Home" page
+      Then The user is redirect to "https://lola-tech.webflow.io/covid19-our-bench-is-your-bench"
```

Comment:
Modify redirect into redirected - The user is redirect to "https://lola-tech.webflow.io/covid19-our-bench-is-your-bench"
Note: please modify everywhere this type of definition